src/settings/components/general-settings/appearance-settings.tsx

```diff
@@ -43,13 +45,18 @@ const AppearanceSettings: FC<{
 		[darkReaderResultValue, items]
 	);
 	const onSelectionChange = useCallback(
-		(v) => {
-			if (DR_VALUES.includes(v) && v !== darkReaderResultValue) {
-				setDarkReaderState(v);
-				addMod('props', 'zappDarkreaderMode', { app: SHELL_APP_ID, value: v });
+		(value) => {
+			if (isDarkReaderPropValues(value)) {
+				if (value !== darkReaderResultValue) {
+					setDarkReaderState(value);
```

Comment:
this set is done twice in both branches of the conditional, just move it out of it :D